content/_data/changelogs/weekly.yml

```
@@ -10730,6 +10730,82 @@
     authors:
       - daniel-beck
 
+- version: '2.288'
+  date: 2021-04-11
+  changes:
+  - type: major bug
```

```suggestion
  - type: bug
```

Even if it is a regression, does not look like a major bug